index.js

```diff
@@ -46,9 +81,12 @@ export class UpdateAPK {
       console.log('RNUpdateAPK::getApkVersionSuccess - outdated based on code, local/remote: ' + RNUpdateAPK.versionCode + "/" + remote.versionCode);
       outdated = true;
     }
-    if (!remote.versionCode && (RNUpdateAPK.versionName !== remote.versionName)) {
-      console.log('RNUpdateAPK::getApkVersionSuccess - APK outdated based on version name, local/remote: ' + RNUpdateAPK.versionName + "/" + remote.versionName);
-      outdated = true;
+    if (!remote.versionCode) {
+      outdated = RNUpdateAPK.versionName !== remote.versionName
+      if(this.options.compareVersionNameByParts){
+        outdated = this.checkVersion(RNUpdateAPK.versionName,remote.versionName) === -1
+      }
```

Comment:
nitpick: fixed spacing
logic comment: shouldn't this be `< 0` to be more correct ?

```suggestion
      if (this.options.compareVersionNameByParts) {
        outdated = this.checkVersion(RNUpdateAPK.versionName,remote.versionName) < 0
      }
```